### PR TITLE
zh:update-/docs/tasks/observability/logs/otel-provider/index.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/docs/tasks/observability/logs/otel-provider/index.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+Envoy 代理可以被配置为以 [OpenTelemetry 格式](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/access_loggers/open_telemetry/v3/logs_service.proto)导出[访问日志](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage)。
+在这个例子中，Envoy 代理将访问日志发送到一个 [OpenTelemetry 收集器](https://github.com/open-telemetry/opentelemetry-collector)，
+该收集器被配置为将日志打印到标准输出。然后可以通过 `kubectl logs` 命令访问 OpenTelemetry 收集器的标准输出。。
+
+{{< boilerplate before-you-begin-egress >}}
+
+{{< boilerplate start-httpbin-service >}}
+
+{{< boilerplate start-otel-collector-service.md >}}
```
Comment (zirain) [inline content/zh/docs/tasks/observability/logs/otel-provider/index.md:18]:
```suggestion
{{< boilerplate start-otel-collector-service >}}
```

File: content/zh/docs/tasks/observability/logs/otel-provider/index.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+{{< text plain >}}
+[%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\" %RESPONSE_CODE% %RESPONSE_FLAGS% %RESPONSE_CODE_DETAILS% %CONNECTION_TERMINATION_DETAILS%
+\"%UPSTREAM_TRANSPORT_FAILURE_REASON%\" %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\"
+\"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\" %UPSTREAM_CLUSTER% %UPSTREAM_LOCAL_ADDRESS% %DOWNSTREAM_LOCAL_ADDRESS% %DOWNSTREAM_REMOTE_ADDRESS% %REQUESTED_SERVER_NAME% %ROUTE_NAME%\n
+{{< /text >}}
+
+下表显示了一个使用默认访问日志格式的例子，即从 `sleep` 发送到 `httpbin` 的请求：
+
+| Log operator | access log in sleep | access log in httpbin |
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/logs/otel-provider/index.md:123]: This line needs translation.

File: content/zh/docs/tasks/observability/logs/otel-provider/index.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    {{< /text >}}
+
+1.  检查 `otel-collector` 的日志：
+
+    {{< text bash >}}
+    $ kubectl logs -l app=otel-collector -n istio-system
+    [2020-11-25T21:26:18.409Z] "GET /status/418 HTTP/1.1" 418 - via_upstream - "-" 0 135 3 1 "-" "curl/7.73.0-DEV" "84961386-6d84-929d-98bd-c5aee93b5c88" "httpbin:8000" "127.0.0.1:80" inbound|8000|| 127.0.0.1:41854 10.44.1.27:80 10.44.1.23:37652 outbound_.8000_._.httpbin.foo.svc.cluster.local default
+    {{< /text >}}
+
+注意，与请求相对应的信息会出现在源和目标的 Istio 代理的日志中，分别是 `sleep` 和 `httpbin`。你可以在日志中看到 HTTP 动词（`GET`）、HTTP路径（`/status/418`）、响应代码（`418`）和其他与[请求相关的信息](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#format-rules)。
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/logs/otel-provider/index.md:176]:
```suggestion
注意，与请求相对应的信息会出现在源和目标的 Istio 代理的日志中，分别是 `sleep` 和 `httpbin`。
您可以在日志中看到 HTTP 动词（`GET`）、HTTP 路径（`/status/418`）、响应代码（`418`）和其他与[请求相关的信息](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage#format-rules)。
```

File: content/zh/docs/tasks/observability/logs/otel-provider/index.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+## 清理{#cleanup}
+
+关闭 [sleep]({{< github_tree >}}/samples/sleep) 和 [httpbin]({{< github_tree >}}/samples/httpbin)服务：
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/logs/otel-provider/index.md:180]:
```suggestion
关闭 [sleep]({{< github_tree >}}/samples/sleep) 和 [httpbin]({{< github_tree >}}/samples/httpbin) 服务：
```

File: content/zh/docs/tasks/observability/logs/otel-provider/index.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+$ kubectl delete -f @samples/httpbin/httpbin.yaml@
+$ kubectl delete -f @samples/open-telemetry/otel.yaml@
+{{< /text >}}
+
+### 禁用 Envoy 的访问日志{#disable-envoy-access-logging}
+
+移除 Istio 安装配置中的 `meshConfig.extensionProviders` 和 `meshConfig.defaultProviders` 设置，或将其设为 `""`。
+
+{{< tip >}}
+在下面的例子中，用你安装 Istio 时使用的配置文件的名称替换 `default` 。
```
Comment (windsonsea) [inline content/zh/docs/tasks/observability/logs/otel-provider/index.md:194]:
```suggestion
在下面的例子中，用您安装 Istio 时使用的配置文件的名称替换 `default`。
```

File: content/zh/docs/tasks/observability/logs/otel-provider/index.md [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+
+{{< text plain >}}
+[%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\" %RESPONSE_CODE% %RESPONSE_FLAGS% %RESPONSE_CODE_DETAILS% %CONNECTION_TERMINATION_DETAILS%
+\"%UPSTREAM_TRANSPORT_FAILURE_REASON%\" %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\"
+\"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\" %UPSTREAM_CLUSTER% %UPSTREAM_LOCAL_ADDRESS% %DOWNSTREAM_LOCAL_ADDRESS% %DOWNSTREAM_REMOTE_ADDRESS% %REQUESTED_SERVER_NAME% %ROUTE_NAME%\n
+{{< /text >}}
+
+下表显示了一个使用默认访问日志格式的例子，即从 `sleep` 发送到 `httpbin` 的请求：
+
+| Log operator | access log in sleep | access log in httpbin |
```
Comment (yanrongshi) [inline content/zh/docs/tasks/observability/logs/otel-provider/index.md:123]:
```suggestion
|   日志运算符   |       访问登录睡眠    |    httpbin 访问登录    |
```

**Comment by yanrongshi** [comment]
I don't know why my CLA validation didn't pass, so I'm going to create a new branch to open this commit

**Comment by windsonsea** [comment]
@zirain Would you like to rescan your cla? I guess you have changed your email recently, so the cla is failed.
https://github.com/istio/istio.io/pull/11564/checks?check_run_id=8279982386

**Comment by yanrongshi** [comment]
Fix in #11883

**Pull request closed without merging**